django/contrib/gis/static/gis/js/OLMapWidget.js

```diff
@@ -1,378 +1,385 @@
 /*global OpenLayers*/
-/*eslint indent:1*/
```

Comment:
@treyhunner, wasn't sure if you had something else in mind for the indentation of this file?